frontends/web/src/routes/account/send/send.tsx

```diff
@@ -158,28 +158,10 @@ class Send extends Component<Props, State> {
     });
 
     this.unsubscribeList = [
-      apiWebsocket((payload) => {
-        if ('type' in payload) {
-          const { data, meta, type } = payload;
-          switch (type) {
-          case 'device':
-            switch (data) {
-            case 'signProgress':
-              this.setState({ signProgress: meta, signConfirm: false });
-              break;
-            case 'signConfirm':
-              this.setState({ signConfirm: true });
-              break;
-            }
-            break;
-          }
-        }
-      }),
```

Comment:
Can you type these 2 subscriptions in api/devicessync?

It should be similar to channelHashChanged in api/devicessync.ts